inbox/react/migration-guide.mdx

```diff
@@ -0,0 +1,737 @@
+  IMessage,
+  MessageActionStatusEnum,
+  useUpdateAction,
+  ButtonTypeEnum,
+  NotificationBell,
+} from "@novu/notification-center";
+
+export default function Novu() {
+
+  const CustomNotificationCenter = () => {
+    const { updateAction } = useUpdateAction();
+
+    const handleOnActionClick = async (
+      templateIdentifier: string,
+      btnType: ButtonTypeEnum,
+      notification: IMessage
+    ) => {
+      if (templateIdentifier === "friend-request") {
+        if (btnType === "primary") {
+          /** Call your API to accept the friend request here **/
+
+          /** And then update Novu that this action has been taken, so the user won't see the button again **/
+          updateAction({
+            messageId: notification._id,
+            actionButtonType: btnType,
+            status: MessageActionStatusEnum.DONE,
+          });
+        }
+      }
+    };
+
+    return (
+      <PopoverNotificationCenter
+        colorScheme={"dark"}
+        onActionClick={handleOnActionClick}
+      >
+        {({ unseenCount }) => <NotificationBell unseenCount={unseenCount} />}
+      </PopoverNotificationCenter>
+    );
+  };
+
+  return (
+    <NovuProvider
+      subscriberId={"SUBSCRIBER_ID"}
+      applicationIdentifier={"APPLICATION_IDENTIFIER"}
+    >
+      <CustomNotificationCenter />
+    </NovuProvider>
+  );
+}
+```
+
+- New Implementation with @novu/react
+
+```tsx
+import { Inbox } from "@novu/react";
+
+function Novu() {
+  const handlePrimaryActionClick = (notification) => {
+    // Handle primary action
+    console.log("Primary action clicked:", notification);
+  };
+
+  const handleSecondaryActionClick = (notification) => {
+    // Handle secondary action
+    console.log("Secondary action clicked:", notification);
+  };
+
+  return (
+    <Inbox
+      applicationIdentifier="YOUR_APP_ID"
+      subscriberId="YOUR_SUBSCRIBER_ID"
+      onPrimaryActionClick={handlePrimaryActionClick}
+      onSecondaryActionClick={handleSecondaryActionClick}
+    />
+  );
+}
+
+export default Novu;
+```
+
+## Avatar Icons
+
+Customize the avatar displayed alongside notifications by specifying an avatar `URL` at the workflow definition level using `@novu/framework`.
+
+```ts
+import { workflow } from "@novu/framework";
+
+workflow("welcome-notification", async (step) => {
+  await step.inApp("inbox", async () => ({
+    subject: "Welcome to Our Service!",
+    body: "We are thrilled to have you onboard.",
+    avatar: "https://example.com/path-to-your-avatar-image.png",
+  }));
+});
+```
+
+## Custom Bell Icon
+
+Customize the bell icon that triggers the notifications popover using the `renderBell` prop.
+
+- Old Implementation
+
+```tsx
+<PopoverNotificationCenter colorScheme="dark">
+  {({ unseenCount }) => (
+    <CustomBell
+      unseenCount={unseenCount}
+      colorScheme="dark"
+      unseenBadgeBackgroundColor="#FFFFFF"
+      unseenBadgeColor="#FF0000"
+    />
+  )}
+</PopoverNotificationCenter>
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** please show the full example with wrapping into the component